core/fuzz/fuzz_targets/block_read_v2.rs

```diff
@@ -1,12 +1,13 @@
 #![no_main]
+use libfuzzer_sys::fuzz_target;
+
 extern crate grin_core;
-#[macro_use]
-extern crate libfuzzer_sys;
 
 use grin_core::core::UntrustedBlock;
-use grin_core::ser;
+use grin_core::ser::{self, DeserializationMode};
 
 fuzz_target!(|data: &[u8]| {
 	let mut d = data.clone();
-	let _t: Result<UntrustedBlock, ser::Error> = ser::deserialize(&mut d, ser::ProtocolVersion(2));
+	let _t: Result<UntrustedBlock, ser::Error> =
+		ser::deserialize(&mut d, ser::ProtocolVersion(2), DeserializationMode::Full);
```

Comment:
ah, we missed these in one of the previous refactorings didn't we? Good this was found and fixed :+1:
Do you know if we run any fuzz testing when making a release or on CI?

Reply:
No, fuzz testing is very manual and I don't think anyone's looked at it in years. I updated all of it because they're small and only took a few minutes to update, so someone can have a look later if they like. If it had been a non-trivial task I would have just removed or disabled them.